Makefile

```diff
@@ -12,6 +12,7 @@ DEIS_PROFILE ?= usw
 DEIS_APP ?= kumascript-dev
 PRIVATE_IMAGE ?= ${PRIVATE_REGISTRY}/${DEIS_APP}\:${VERSION}
 TEST_RUN_ARGS ?=
+TEST_RUN_TIMEOUT ?= 10000
```

Comment:
I changed this timeout setting specifically for the server code tests (``make test``) rather than the macro tests (``make test-macros``). None of the macro tests currently make real network calls (e.g., ``test-httpheader.js`` stubs the ``wiki.getPage`` calls). I had debated making separate timeouts for each, but decided we could do that later if we thought it best.